### PR TITLE
Style output, add xdebug shortcut

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

## XDebug is installed 

```bash
./yii serve --xdebug
```

image

```bash
./yii serve
```

image

## XDebug is not installed / loaded 

When `xdebug` extension is not installed / loaded
```bash
./yii serve
```
same for `--xdebug`

image

## Verbose

```bash
./yii serve --xdebug -v
```
image

```bash
./yii serve -v
```
image

### CODE REVIEW

File: src/Command/Serve.php [hunk 1]
```diff
@@ -79,6 +94,8 @@ public function complete(CompletionInput $input, CompletionSuggestions $suggesti
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
+        $io->title('Yii 3 Development Server.');
```
Comment (vjik) [inline src/Command/Serve.php:97]:
```suggestion
        $io->title('Yii3 Development Server');
```

File: src/Command/Serve.php [hunk 2]
```diff
@@ -79,6 +94,8 @@ public function complete(CompletionInput $input, CompletionSuggestions $suggesti
...
+        $io->writeln('https://yiiframework.com');
```
Comment (vjik) [inline src/Command/Serve.php:98]:
```suggestion
        $io->writeln('https://yiiframework.com'."\n");
```

**Review by vjik — CHANGES_REQUESTED** [review]
Need line in changelog

File: src/Command/Serve.php (hunk 2 above)
Comment (samdark) [reply src/Command/Serve.php:98]: It is `writeln` already. Do you mean additional line?

**Comment by samdark** [comment]
Tests should be fixed.

File: src/Command/Serve.php [hunk 3]
```diff
@@ -122,27 +141,55 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
 
-        $output->writeLn("Document root is \"$documentRoot\"");
-
-        if ($router) {
-            $output->writeLn("Routing file is \"$router\"");
-        }
-
-        $output->writeLn('Quit the server with CTRL-C or COMMAND-C.');
-
         if ($env === 'test') {
```
Comment (samdark) [inline src/Command/Serve.php:144]: Why suppressing actual tests?

File: src/Command/Serve.php (hunk 2 above)
Comment (vjik) [reply src/Command/Serve.php:98]: Yes, here:
image

File: src/Command/Serve.php (hunk 3 above)
Comment (xepozz) [reply src/Command/Serve.php:144]: It was there, I don't know why

**Review by terabytesoftw — APPROVED** [review]

**Review by vjik — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Pull request merged**